src/controller/CommissionerDiscoveryController.cpp

```diff
@@ -219,6 +219,28 @@ void CommissionerDiscoveryController::InternalOk()
         ChipLogError(AppServer, "UX InternalOk: could not find instance=%s", mCurrentInstance);
         return;
     }
+
+    bool isContentAppInstalled = mAppInstallationService->HasContentApp(client->GetVendorId(), client->GetProductId());
+
+    if (!isContentAppInstalled) {
+        ChipLogDetail(AppServer, "UX InternalOk: app not installed.");
+
+        // TODO: send CDC message that user is prompted to install the app
+
```

Comment:
This would be a CommissionerDeclaration message with the kAppInstallConsentPending error code set.

Reply:
Added.